### DISCUSSION TITLE
Create exportable shortcut profile

### DISCUSSION THREAD

**Discussion by Fhining** [SubtitleEdit/subtitleedit#7924] [Ideas]
Hello! Hope everyone's ok.

I wanted to suggest or ask if there's a way to export a profile that contains your shortcut configuration.

I understand that you can do this on the portable version, but it would be great if we could do the same like the "Subtitle Profile" export- import function.

The reason is that if you want to send someone your shortcut configuration and both aren't using the portable version, there's no way to do this. As I understand, you can do it sending the "Settings" files from the Portable version.

This would help a lot on keeping your shortcuts safe, in case you need to reinstall SE or move to another PC.

Thank you so much! :)

EDIT: Sorry for creating this topic. I'm trying to delete this post but I can't. I just saw the post for "EXPORT" option doing a right click on the screen.

**Comment by sherkhan30452** [comment]
You (since you are the one created) can close this discussion instead.
